wire-ios/Wire-iOS/Sources/UserInterface/GroupDetails/UICollectionView+GroupedSections.swift

```diff
@@ -29,13 +29,18 @@ fileprivate extension UICollectionViewFlowLayout {
 }
 
 extension UICollectionView {
-    convenience init(forGroupedSections: ()) {
+    convenience init(forGroupedSections: (), usedInMessageDetailsVC: Bool = false) {
         self.init(frame: .zero, collectionViewLayout: UICollectionViewFlowLayout(forGroupedSections: ()))
         backgroundColor = .clear
         allowsMultipleSelection = false
         keyboardDismissMode = .onDrag
         bounces = true
         alwaysBounceVertical = true
-        contentInset = UIEdgeInsets(top: 32, left: 0, bottom: 32, right: 0)
+        if usedInMessageDetailsVC {
+            contentInset = UIEdgeInsets(top: 12, left: 0, bottom: 32, right: 0)
+        } else {
+            contentInset = UIEdgeInsets(top: 32, left: 0, bottom: 32, right: 0)
+        }
+
```

Comment:
Why is this needed?

Reply:
This is needed so we change the top inset for our header in MessageDetails to match the design specs. I wanted to avoid changing it in all places and that's why I introduced that property.

Reply:
Can it be set externally? Something like:

```swift
let collectionView = ...
collectionView.contentInset = ...
```

Reply:
Done!